src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -39,6 +46,24 @@ public void deleteCart(@PathVariable Long cartId) {
         service.deleteCart(cartId);
     }
 
+    @GetMapping("/item/{itemId}")
+    public ItemDto getItem(@PathVariable Long itemId) {
+        return mapper.mapToItemDto(service.getItem(itemId));
+    }
 
+    @PostMapping(path ="/newItem", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public void addItem(@RequestBody ItemDto itemDto){
+        service.addItem(mapper.mapToItem(itemDto));
+    }
+
+    @DeleteMapping("/item/{itemId}")
```

Comment:
Sugestia - może samo id

Reply:
Skoro to osobny endpoint dla itemu wewnątrz Cart to można zostawić w ten sposób